app-ios/Sources/SearchFeature/DayFilterSheetView.swift

```diff
@@ -0,0 +1,100 @@
+import appioscombined
+import Assets
+import Model
+import SwiftUI
+import Theme
+
+private extension DateFormatter {
+    static let eventDateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.calendar = Calendar(identifier: .gregorian)
+        formatter.dateStyle = .long
+        formatter.dateFormat = "YYYY EEEE d"
+        return formatter
+    }()
+}
+
+private extension DroidKaigi2022Day {
+    // NOTE: This implementation is not good, but it is acceptable for this use case.
+    func eventDate() -> String {
+        let eventDateFormatter = DateFormatter.eventDateFormatter
+        return "\(name) (\(eventDateFormatter.string(from: start.toDate()))th)"
```

Comment:
This looks strange for Japanese users.

```suggestion
        return "\(name) (\(eventDateFormatter.string(from: start.toDate())))"
```